<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Png/icon.png</normaloff>Png/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #292929;</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>14</y>
     <width>358</width>
     <height>365</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> /*Виділення по натиску*/

*{color: white}

QListView::item:selected:active {
                background-color: #ff7f00;
                color: black;
                font-size: 93%;
            }
 /*Обичне виділення*/
            QListView::item:selected:!active {
                background-color: #FFA500;
                color: white;
                font-size: 93%;
            }</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>385</y>
     <width>134</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Greate new playlist</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #808080;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Png/icon.png</normaloff>Png/icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_2">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>385</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Add a song</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #808080;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Png/icon.png</normaloff>Png/icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_3">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>385</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Delete the song</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #808080;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Png/icon.png</normaloff>Png/icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
